2023/did-wg.html

```diff
@@ -240,34 +234,18 @@ <h3>
                   <b>Exclusion Draft Charter</b>: <a href="https://www.w3.org/2020/12/did-wg-charter.html">https://www.w3.org/2020/12/did-wg-charter.html</a>.  
               </p>
             </dd>
-          </dl>
-          <p>
-            The Working Group may begin work on the following W3C normative specifications:
-          </p>
-
-          <dl>
-            <dt class="spec">Decentralized Identifier (DID) Method Specification</dt>
-           <dd>
-             <p>
-             DID Method Specifications may be produced by the working group.
-             </p>
-             <p class="milestone">
-              <b>Expected Completion:</b> It is not intended that any DID Method Specification developed by the working group will advance beyond <a href="https://www.w3.org/Consortium/Process/#RecsWD">Working Draft status</a>.
-             </p>
-             <p>
-             <b>Adopted Draft:</b> The <a href="https://www.w3.org/TR/did-spec-registries/#did-methods">DID Specification Registries</a> contains many links to examples of DID Method Specifications, some of which may be selected by the working group for development into a Recommendation. The group might also choose a similar starting document not listed in the Registry.
-             </p>
-           </dd>
+           </dl>
+           <dl id="deliverable-did-resolution">
             <dt class="spec">Decentralized Identifier (DID) Resolution and DID URL Dereferencing v1.0</dt>
-           <dd>
+            <dd>
              <p>
              This document specifies the algorithms and guidelines for resolving DIDs and dereferencing DID URLs.
              </p>
              <p class="draft-status">
               <b>Draft State:</b> Draft Community Group Report
              </p>
              <p class="milestone"><b>Expected completion:</b>
-             It is not intended that DID Resolution and DID URL Dereferencing will advance beyond <a href="https://www.w3.org/Consortium/Process/#RecsWD">Working Draft status</a>.
+             <i class="todo">START+24M</i>.
```

Comment:
```suggestion
             <i class="todo">START+36M</i>.
```
I don't believe we can get through this work in two years.

Reply:
Just an admin remark: if this change is approved and merged, the "metadata" at the beginning of the charter proposal should also be changed (it says, for the end date, "Start date + 2 years").

However, note that, these days, the management is reluctant to allow a charter for three years. The WG might be prepared to say, instead, that it asks for two years with a possibility for an extension at the end of that period; essentially, giving the AC the possibility for a check on whether the WG is really progressing. Something to be discussed in advance...

Reply:
Thanks @iherman, I have nothing to add. @jandrieu, @peacekeeper could you live with that (i.e. keeping the 24M deadline in the proposed charter)?

Reply:
Two years with possibility for an extension sounds good to me.

Reply:
Two years with possibility for an extension sounds okay to me.